src/worker.cc

```diff
@@ -94,7 +94,7 @@ const std::unordered_map<unsigned, unsigned> ERROR_TO_STATUS{
 
     {120, 400}, {121, 400}, {122, 400}, {123, 400}, {124, 400}, {125, 400}, {126, 400}, {127, 400},
 
-    {130, 400}, {131, 400}, {132, 400}, {133, 400}, {136, 400},
+    {130, 400}, {131, 400}, {132, 400}, {133, 400}, {136, 400}, {137, 400},
```

Comment:
you need to also add mappings to the OSRM flavored errors below

Reply:
also why isnt there a new entry for 167? its already got a mapping below but it looks new in the header file :thinking:

Reply:
I think that's a (unnoticed) "artifact" from the shortcuts-to-bins PR